clients/rospy/src/rospy/impl/tcpros_service.py

```diff
@@ -136,7 +136,7 @@ def contact_service(resolved_name, timeout=10.0):
             except: # service not actually up
                 if first:
                     first = False
-                    rospy.core.logerr("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
+                    rospy.core.loginfo("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
```

Comment:
This message would be printed every 0.3 s. I think that is too verbose. Please consider using a throttled log call (http://wiki.ros.org/rospy/Overview/Logging#Logging_Periodically). Maybe once every 10s is ok?